app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,28 @@
+!!!
+%html
+  %head
+    %title Instagram
+    = stylesheet_link_tag 'application', media: 'all',
+    'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+    %meta(charset="UTF-8")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))